tasties_app/models.py

```diff
@@ -26,6 +38,14 @@ class Recipe(models.Model):
     minutes_to_make = models.IntegerField(validators=[MinValueValidator(1)])
     recipe_picture = models.ImageField(upload_to="recipe_pictures")
 
+    def __str__(self):
+        user_name = self.author_id.username
+        string_categories = ""
+        for i in self.categories.all():
+            string_categories = string_categories + str(i.category_name)
+        return user_name + " " + string_categories + " " + self.title + " " + self.description + " " + self.directions \
```

Comment:
Is this how we want the recipe to be presented? I think the title would be enough, maybe with the description too.

Reply:
I think if you just want to do: 
print(recipe)
it should print the title,
but of course, when we display the recipe on the app it will include all the details.

Reply:
@GilShechter, I fix like you suggest.